tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/RetryTest.java

```diff
@@ -126,6 +128,37 @@ public void testRetryWithDelay() {
         Assert.assertTrue(clientForDelay.isDelayInRange(), "The delay between each retry should be 0-800ms");
     }
 
+
+    /**
+     * Testing whether the {@code @Retry} annotation on method serviceB overrides the Class level
+     * {@code @Retry} annotation.
+     *
+     * Delay is 0 and jitter 400ms. Invocation takes 3200ms and effective delay must be between 0 and 400ms.
+     */
+    @Test
+    public void testRetryWithNoDelayAndJitter() {
+        try {
+            retryClientWithNoDelayAndJiter.serviceA();
+            Assert.fail("serviceA should throw a RuntimeException in testRetryWithDelay");
+        }
+        catch (RuntimeException ex) {
+            // Expected
+        }
+
+        final int retryCountForConnectionService = retryClientWithNoDelayAndJiter.getRetryCountForConnectionService();
+        final int fastDelays = retryClientWithNoDelayAndJiter.fastDelays();
+
+        // 60% to discard false positives
+        Assert.assertTrue(fastDelays < (retryCountForConnectionService * 0.6),
```

Comment:
I think there's a fairly high chance that this test will fail due to random chance, assuming that the jitter is evenly distributed between -400ms and 400ms and then clamped to be above zero (maybe about 20% of the time, depending on how many retry attempts are needed)

Reply:
On the other hand, if you're expecting the jitter to be evenly distributed between 0 and 400ms, then you're probably safe.

Reply:
Yes, there some risk here. My Idea is that jitter is evenly distributed between -400ms and 400ms but all values <0 will be 0. That leads to ~50% of the values being between  0 and 400ms. The 60% means that, 40% of the values will be between 5ms and 400ms... This can be fine tuned.